File: C/program60.c
#include<stdio.h>


void DisplayFactor(int iNo)
{
    if(iNo < 0)
    {
        iNo = - iNo;
    }

    int iCnt = 0;

    printf("Fcators are :\n");
    for(iCnt = 1; iCnt < iNo/2; iCnt++)
    {
        if((iNo % iCnt) == 0)
        {
            printf("%d\n",iCnt);
        }
    }
}

int main()
{
    int iValue = 0;
   
    printf("Enter number :");
    scanf("%d",&iValue);

    DisplayFactor(iValue);

    return 0;
}

File: Java/program60.java
import java.util.*;
 
 class Number
 {
    public  void DisplayFactor(int iNo)
    {
        if(iNo < 0)
        {
            iNo = -iNo;
        }
        int iCnt = 0;
        for(iCnt = 1; iCnt <= iNo / 2; iCnt++)
        {
            if((iNo % iCnt) == 0)
            {
                System.out.println(iCnt);
            }
        }
    }
 }
class program60
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        

        System.out.println("Enter number : ");
        int iValue = sobj.nextInt();
      


        Number eobj =new Number();

        eobj.DisplayFactor(iValue);

       
    }
}